reference/best_practices/common_best_practices.rst

```diff
@@ -0,0 +1,28 @@
+Common Best Practices
+=====================
+
+The Akeneo PIM standard project comes with a default configuration to help you during the development of your project.
```

Comment:
You use twice the "project" word in your sentence. You can end it directly after "development".